### PR TITLE
feat: add contirbutor over time image in team page

### PR DESCRIPTION
**Pull request by LiteSun**
Changes:

feat: add contributor over time image in team page

Screenshots of the change:
image

### CODE REVIEW

File: website/src/pages/team.js [hunk 1]
```diff
@@ -290,6 +290,8 @@ function Team(props) {
         list.
       </SectionSubtitle>
       <RepoCardsContainer>{repoComponents}</RepoCardsContainer>
+      <SectionTitle>Contributor Over Time</SectionTitle>
```
Comment (juzhiyuan) [inline website/src/pages/team.js:293]: You need a subtitle to tell visitors this flow contains all repos.

**Review by Yiyiyimu — APPROVED** [review]

**Comment by iamayushdas** [comment]
@LiteSun any update on this PR?

File: website/src/pages/team.js (hunk 1 above)
Comment (LiteSun) [reply website/src/pages/team.js:293]: ok

**Comment by iamayushdas** [comment]
if you are busy @LiteSun should i push a PR on this?
scrnli_12_05_2021_18-12-43

**Comment by LiteSun** [comment]
> if you are busy @LiteSun should i push a PR on this?
> scrnli_12_05_2021_18-12-43

Sure, welcome.

**Comment by iamayushdas** [comment]
Okay then

**Comment by LiteSun** [comment]
ping @juzhiyuan

**Review by juzhiyuan — APPROVED** [review]

**Comment by LiteSun** [comment]
cc @moonming

**Pull request merged**